NEWS

```diff
@@ -11,9 +11,98 @@
 ### Features:
 ### Bugfixes:
 
-## 1.17.0-rc1 (April 19)
-TBD
-
+## 1.17.0-rc2 (May 16, 2024)
+### Features:
+#### UCP
+* Improved performance calculation for rendezvous protocol
+* Enabled short protocol for non-host memory types on empty messages
+* Improved performance estimation for empty messages by removing non-relevant overheads
+* Added RMA_ZCOPY_MAX_SEG_SIZE configuration parameter to allow modifying segment size for RMA-ZCOPY protocols
+* Added support for separate intra/inter-node rendezvous thresholds
+* Added support for minimal fragment size in rendezvous protocol
+* Added support for protocol variants
+* Added support for resetting request during send operation
+* Added UCX_PROTO_OVERHEAD configuration variable to allow setting protocol overheads
+* Improved performance for combined Active Message/RMA scenarios by separating them to different lanes
+* Added support for device staging buffers in pipeline protocols
+* Enable ODP on Grace platforms by default 
+#### RDMA CORE (IB, ROCE, etc.)
+* Introduced the UCX_REVERSE_SL environment variable to configure reverse SL for DC transport. By default, it uses UCX_IB_SL.
+* Added support for GID auto-detection in Floating LID based routing
+* Imporved performance of RC transport when hardware tag matching is not available
+* Added support for multithreading KSM registration of unaligned buffers
+* Added IB_SEND_OVERHEAD and MM_[SEND|RECV]_OVERHEAD configuration variables
+#### GPU (CUDA, ROCM)
+* Added support for oneAPI Level-Zero library for intel GPUs
+#### UCS
+* Added support for rcache dynamic region alignment
+* Added dynamic bitmap data structure
+* Added support for advanced key-value parsing for UCX configuration
+* Added piecewise linear function data structure
+* Added support for allocating dynamic arrays on stack
+#### Tests
+* Added wire compatibility tests for v1.16 
+#### Tools
+* Added support for device memory allocation in UCX perftest
+* Added a script to use for squashing commits after PR approval
+* Added support for DPU cross-gvmi daemon in UCX perftest
+#### Java
+* Added support for EP local socket address API in JUCX
+#### Build
+* Added Address Sanitizer support
+* Added a helper shell script to run static checks
+#### AZP
+* Replaced Valgrind tests by Address Sanitizer 
+* Added Ubuntu 22.04 docker image testing
+* Updated gdrcopy module version
+#### Configuration
+* Added support for filtering configuration sections by platform type
+* Added configuration file with section for Grace Hopper
```

Comment:
pls add emplty line below

Reply:
seems in 1.16 there's no space line (line 169)

Reply:
ok